mico-admin/src/app/api/semantic-version.ts

```diff
@@ -74,15 +113,11 @@ export function incrementVersion(version: String) {
 
     // patch
     if (match[4] != null) {
-        versionString += match[4];
-    } else {
-        versionString += '0';
-    }
+        // since no value was returned yet, the incrementComponent must be 'patch' or some kind of undefined.
+        versionString += (parseInt(match[4], 10) + 1).toString();
 
-    // optional letters in the end
-    if (match[5] != null) {
-        versionString += match[5];
+        return versionString + versionAppendix;
     }
 
-    return versionString;
+    return undefined;
```

Comment:
add a patch version of 1 if no patch version was present before instead of returning undefined